statistics/histogram.go

```diff
@@ -139,8 +139,11 @@ func (hg *Histogram) MemoryUsage() (sum int64) {
 	if hg == nil {
 		return
 	}
+	if len(hg.Buckets) == 0 && len(hg.scalars) == 0 && hg.Bounds.Capacity() == 0 {
+		return
+	}
```

Comment:
Empty histogram consumes memory. Is there a way to track and evict the empty part too?

Reply:
We use empty histograms in order to avoid panic problem. Thinking empty histograms as 0 would be friendly to our lru cache as finally tracking cost can be 0. I think the difference of the memory cost for empty histogram and zero is accptable.

Reply:
It's fine for most of the cases, while for some cases with many tables(more than 100,000), let's consider it in another topic.